patterns/roks/variables.tf

```diff
@@ -199,6 +199,12 @@ variable "disable_outbound_traffic_protection" {
   default     = false
 }
 
+variable "force_delete_storage" {
```

Comment:
Might be more user-friendly if we called the input variable something like `cluster_force_delete_storage` to be clear that amongst all these inputs, this one pertains to clusters only? Do not need to do that in the cluster maps or json, just these inputs that would show up in a form on projects or schematics.

Reply:
Ah, this is where we should put the info about deleting the persistent storage.

description = "Whether to delete persistent storage when the associated VPC cluster is deleted so that it can't be recovered. Set to `true` to force the removal of persistent storage. Set to `false` to ...?"